kategory/src/main/kotlin/kategory/instances/OptionInstances.kt

```diff
@@ -81,3 +81,12 @@ interface OptionInstances :
             fa.ev().orElse({ f(Unit).ev() })
 
 }
+
+/**
+ * Dummy SemigroupK instance to be able to test laws for SemigroupK.
+ */
+class OptionSemigroupK : SemigroupK<Option.F> {
+    override fun <A> combineK(x: HK<Option.F, A>, y: HK<Option.F, A>): HK<Option.F, A> {
```

Comment:
Tried doing it but then I can't compile anymore. The method is not recognized as a real override of:

`fun <A> combineK(x: HK<F, A>, y: HK<F, A>): HK<F, A>`

It feels like the typealias is not properly working for this case.

![captura de pantalla 2017-07-27 a las 18 17 48](https://user-images.githubusercontent.com/6547526/28680872-f18b7e24-72f7-11e7-8a29-dda08c13eeee.png)